setup.py

```diff
@@ -57,6 +57,7 @@ def get_version(filename):
         'networkx<=2.2;python_version<"3.5"',
         'junit-xml~=1.9',
         'pyrsistent<=0.16.0;python_version<"3.5"',
+        'future'
```

Comment:
where is this coming from?

Reply:
To resolve the compatibility issue with Python 2.7: ImportError: No module named builtins. The module builtins is used to mock the open() function in the unit tests

Reply:
Weird.  Thought we were doing some tests with the open that didn't require this.  I'm okay with this now but would like to see if we can figure out a better way to do this.